### ISSUE TITLE
How to implement CassandraTable with an abstract model?

### ISSUE THREAD

**Issue by peter-empen** [outworkers/phantom#663]
Cassandra tables may contain some discriminator column that is read first in fromRow to decide on the the concrete Scala representation of a given row.
The following boiled-down example works with Phantom 1.x  but I get "could not find implicit value for parameter helper..." with Phantom 2.6.5:
```
import com.outworkers.phantom.dsl._

sealed trait Base
case class A(a: Int) extends Base
case class B(b: Int) extends Base

class Schema extends CassandraTable[Schema, Base] {
  object discriminator extends IntColumn(this) with PartitionKey
  object a             extends IntColumn(this)
  object b             extends IntColumn(this)
 
  override def fromRow(row: Row): Base =
    if (discriminator(row) % 2 == 0) A(a(row))
    else                             B(b(row))
}
```
How can I get a trait-based model work with Phantom 2?

**Comment by alexflav23** [comment]
Fix in #667, live in Maven Central in 2 hours.

**Comment by peter-empen** [comment]
Thank you for resolving
